src/main/scala/com/actian/spark_vector/vector/VectorConnectionProperties.scala

```diff
@@ -15,19 +15,57 @@
  */
 package com.actian.spark_vector.vector
 
+
+trait JDBCPort extends Serializable {
+    def value: String
+
+    override def toString = value
+}
+
+object JDBCPort {
```

Comment:
Would it make sense to make the regular expressions used below members of the object?